### ISSUE TITLE
Error in BaseBEVBackbone

### ISSUE THREAD

**Issue by MengAaron** [open-mmlab/OpenPCDet#522]
if len(ups) > 1:
   x = torch.cat(ups, dim=1)
for some special case, if input feature map size is [199,199] in basic setting in second, ups[0] and ups[1] may have different size,[199,199] and [200,200] respectively, that may cause error

**Comment by MengAaron** [comment]
to reproduce 199, you can just delete the last layer in VoxelBackBone8x, and set voxel size 0.1,0.1
